components/tooltip/README.md

```diff
@@ -151,3 +151,44 @@ In the following example to constrain the tooltip to the dashed boundary we can
   </d2l-tooltip>
 </div>
 ```
+
+## Help Tooltip [d2l-tooltip-help]
+
+The `d2l-tooltip-help` component is used to display additional information when users focus or hover over some text formatted with an underline.
+
+<!-- docs: demo code name:d2l-tooltip-help autoSize:false size:small -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/tooltip/tooltip-help.js';
+</script>
+
+<d2l-tooltip-help text="Helpful label">Contents should elaborate on the label (be short and concise)</d2l-tooltip-help>
+```
+
+<!-- docs: start hidden content -->
+### Properties
+
+| Property | Type | Description |
+|--|--|--|
+| `text` | String, required | Provide the `text` that will render as the Help Tooltip opener.|
+
+### Events
+* `d2l-tooltip-show`: dispatched when the opener text is hovered over, focused on, or clicked
```

Comment:
Does this component actually dispatch these events or are those more the `d2l-tooltip`'s events?

Reply:
I wouldn't expect this component to dispatch any events unless a use case comes up where they're needed.

Reply:
I'll remove any documentation for events for this component, since we don't want consumers to rely on that fact, and as mentioned, there shouldn't currently be a use case for those events.